docs/serve.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
could we run it on the same port as docusaurus, e.g. 3000? (instead of 5000)

Reply:
Assuming we change to `netlify-cli`, yes we can supply a `-p` flag to set the port

Reply:
But IMO it's helpful to use Netlify's default port of `8888`, because that way I never forget that I'm looking at a built version as opposed to live dev

Reply:
ok so maybe use the same port everywhere (and change docusaurus one + the README)?

Reply:
No I mean rather that in practice I find it more clear to have different ports. When working with Gatsby for example, the port is always `localhost:8000` but the Netlify CLI always serves on `localhost:8888`, and this helps me know from a glance what I'm looking at. I think if Docusaurus serves on `5000` it's clear whether we're looking at development mode or production mode, just from the URL